src/tests/topp/FalseDiscoveryRate_output_4.idXML

```diff
@@ -2179,7 +2179,7 @@
 			</PeptideHit>
 		</PeptideIdentification>
 		<PeptideIdentification score_type="q-value" higher_score_better="false" significance_threshold="0" MZ="539.28" RT="990" >
-			<PeptideHit score="0.121212121212121" sequence="LLDEVEELLEQER" charge="3" aa_before="R" aa_after="L" protein_refs="PH_9 PH_10 PH_11 PH_12 PH_464" >
+			<PeptideHit score="0.121212121212121" sequence="LLDEVEELLEQER" charge="3" aa_before="R X X X X" aa_after="L X X X X" protein_refs="PH_9 PH_10 PH_11 PH_12 PH_464" >
```

Comment:
That is how it was used before. Only for the first peptide evidence (first match to a protein) this information was available in idXML (for whatever reason). Newer idXML should contain this information for every hit. I adapted the TandemXML file parser but full support will probably be added step by step.